mypyc/genops.py

```diff
@@ -2863,15 +2962,17 @@ def add_call_to_callable_class(self,
         return call_fn_ir
 
     def instantiate_callable_class(self, fn_info: FuncInfo) -> Value:
-        """Assigns a callable class to a register named after the given function definition."""
+        """Assigns a callable class to a register named after the given function definition. Note
+        that fn_info refers to the function being assigned, whereas self.fn_info refers to the
+        function encapsulating the function being turned into a callable class.
```

Comment:
Nit: blank line after the first one line summary.

Reply:
Done.